Resources/Textures/_Impstation/Objects/Storage/littlecar.rsi/meta.json

```diff
@@ -0,0 +1,19 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "copyright": "TODO, for Impstation.",
```

Comment:
Rahhhhhh

Reply:
It should be PR author's name (hopefully correct) plus a link to the merged commit since it's CC BY-SA.

Reply:
So:
```suggestion
    "copyright": "widgetbeck via https://www.github.com/impstation/imp-station-14/pull/439",
```
?

Reply:
ye